UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -34,10 +36,18 @@ public class EnergyTests : EnergyTestsBase
 
         protected override double CaloriesInOneJoule => 0.239005736;
 
+        protected override double DecaeuThermsInOneJoule => 9.4781339449889105832843629746176e-10;
+
+        protected override double DecaimperialThermsInOneJoule => 9.478171203551087813109937767482e-15;
+
+        protected override double DecausThermsInOneJoule => 9.4804342797334860315281322406817e-10;
+
         protected override double ElectronVoltsInOneJoule => 6.241509343260179e18;
 
         protected override double ErgsInOneJoule => 10000000;
 
+        protected override double EuThermsInOneJoule => 9.4781339449889105832843629746176e-9;
```

Comment:
I get this to 9.47816988e-9 when doing 1/105505600 in [Google](https://www.google.se/search?dcr=0&source=hp&q=1%2F105505600&oq=1%2F105505600)

Reply:
Yeah, make sense, I'll change this value.